go/cmd/dolt/commands/sqlserver/serverconfig.go

```diff
@@ -25,140 +27,221 @@ type LogLevel string
 var CliVersion = "test"
 
 const (
+	LogLevel_Trace   LogLevel = "trace"
 	LogLevel_Debug   LogLevel = "debug"
 	LogLevel_Info    LogLevel = "info"
 	LogLevel_Warning LogLevel = "warning"
 	LogLevel_Error   LogLevel = "error"
 	LogLevel_Fatal   LogLevel = "fatal"
 )
 
+// String returns the string representation of the log level.
+func (level LogLevel) String() string {
+	switch level {
+	case LogLevel_Trace:
```

Comment:
Can't you combine all these into one case statement?

Reply:
Yeah, I was just matching style and going for a minimal change... but I do hate the way this is written.